java-logging-appinsights/README.md

```diff
@@ -146,25 +156,28 @@ spring:
     name: My Application Insights WebApp
 ```
 
-This configuration entry is also used by [`CloudRoleNameInitializer`](src/main/java/uk/gov/hmcts/reform/logging/appinsights/telemetry/initializers/CloudRoleNameInitializer.java) to set the `cloud_RoleName` App Insights tag. This makes distinguishing between services easier in cases where a number of related services uses the same App Insights instance.
+This configuration entry is also used by [`SpringBootTelemetryInitializer`](https://github.com/Microsoft/ApplicationInsights-Java/blob/master/azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/initializer/SpringBootTelemetryInitializer.java) to set the `cloud_Role` App Insights tag. This makes distinguishing between services easier in cases where a number of related services uses the same App Insights instance.
```

Comment:
Can this be pointed to specific version too?

Reply:
This section is not strictly true anymore, the spring boot starter defaults to `application`

I wonder if its worth removing it as it doesn't give a lot of value, maybe say, 
Set `spring.application.name` so that you can differentiate multiple applications in the same App Insights instance